### PR TITLE
W-16555774 Get Started Kondo UI Update

### PR DESCRIPTION
**Pull request by andrewyu-salesforce**
# Critical Changes

# Changes
Update NPSP Get Started with NPSP page to be Kondo compatible
# Issues Closed

# Community Ideas Delivered

# Features Intended for Future Release

# Features for Elevate Customers

# New Metadata

# Deleted Metadata

### CODE REVIEW

**Comment by andrewyu-salesforce** [comment]
Screenshot 2024-09-24 at 3 19 57 PM
Screenshot 2024-09-24 at 3 19 49 PM

**Comment by andrewyu-salesforce** [comment]
https://gus.lightning.force.com/lightning/r/ADM_Work__c/a07EE00001zY0hPYAS/view

File: force-app/main/default/flexipages/GetStarted.flexipage-meta.xml [hunk 1]
```diff
@@ -24,7 +24,7 @@
         <name>region1</name>
         <type>Region</type>
     </flexiPageRegions>
-    <flexiPageRegions>
```
Comment (andrewyu-salesforce) [inline force-app/main/default/flexipages/GetStarted.flexipage-meta.xml:27]: This component will never be rendered. Comment it out so the extra spacing can be removed

**Review by force2b — APPROVED** [review]
I pulled up all the changes in a new org with and without Kondo. Looks great!

**Pull request merged**